### PR TITLE
chore: bump version to v4.10.5-beta.5

### PR DESCRIPTION
**Pull request by liruchen32**
# Issue

# Notice

# Dependency

### CODE REVIEW

**Review by Aylie-Chou — CHANGES_REQUESTED** [review]
`yarn.lock` would also need to be updated

**Review by Aylie-Chou — APPROVED** [review]

**Review by Aylie-Chou — CHANGES_REQUESTED** [review]
the ci config change would also need to be included in CHANGELOG.md

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+## 4.10.5-beta.5, 2023-06-15
```
[the remark this answers is not in the document]
Comment (liruchen32) [reply CHANGELOG.md:3]: fixed

[the remark this answers is not in the document]
Comment (Aylie-Chou) [reply CHANGELOG.md:3]: & also commits

[the remark this answers is not in the document]
Comment (liruchen32) [reply CHANGELOG.md:3]: & done

**Review by Aylie-Chou — APPROVED** [review]
LGTM

**Pull request merged**